workflows/maker-annotation/maker-annotation-eukaryote-tests.yml

```diff
@@ -0,0 +1,101 @@
+- doc: Test outline for maker-annotation-eukaryote.ga
+  job:
+    EST and/or cDNA:
+      class: File
+      location: https://zenodo.org/record/5095188/files/cds.fasta?download=1
+    Genome sequence:
+      class: File
+      location: https://zenodo.org/record/5095188/files/genome.fasta?download=1
+    Protein sequences:
+      class: File
+      location: https://zenodo.org/record/5095188/files/proteins.fasta?download=1
+  outputs:
+    BUSCO_full_table_01:
+      class: File
+      location: https://zenodo.org/record/5112538/files/BUSCO_full_table_01.tabular?download=1
+      compare: diff
+      lines_diff: 4
+    BUSCO_missing_orthologs_01:
+      class: File
+      location: https://zenodo.org/record/5112538/files/BUSCO_missing_orthologs_01.tabular?download=1
+      compare: diff
+      lines_diff: 4
+    BUSCO_short_summary_01:
+      class: File
+      location: https://zenodo.org/record/5112538/files/BUSCO_short_summary_01.txt?download=1
+      compare: diff
+      lines_diff: 6
+    BUSCO_short_summary_02:
+      class: File
+      location: https://zenodo.org/record/5112538/files/BUSCO_short_summary_02.txt?download=1
+      compare: diff
+      lines_diff: 6
+    BUSCO_short_summary_03:
+      class: File
+      location: https://zenodo.org/record/5112538/files/BUSCO_short_summary_03.txt?download=1
+      compare: diff
+      lines_diff: 6
+    BUSCO_short_summary_04:
+      class: File
+      location: https://zenodo.org/record/5112538/files/BUSCO_short_summary_04.txt?download=1
+      compare: diff
+      lines_diff: 6
```

Comment:
These all seem to be pretty small, any reason not to include them in the test-data directory ?

Reply:
Not at all, I just decided to include all of them in Zenodo; I'll include the small datasets in the test directory; is it possible to use URL as output for the big ones?

Reply:
Not currently, I am looking at what we need to do for that. Including small output data in the repository has some upsides: You don't need to upload to zenodo, and you can update the test data with the `--update_test_data` flag.

For the big ones it might be a good idea to just include a small snippet in the repo and to use `compare: contains`.